client-common/src/main/java/com/cloudera/livy/client/common/ClientConf.java

```diff
@@ -236,6 +234,18 @@ private void logDeprecationWarning(String key) {
   /** Maps deprecated key to DeprecatedConf with the same key. */
   protected abstract Map<String, DeprecatedConf> getDeprecatedConfigs();
 
+  private Map<String, String> altToNewKeyMap = null;
+  private Map<String, String> allAlternativeKeys() {
```

Comment:
If you could add a comment above this to explain its purpose, it took me some time to figure out what it was doing and it will save future devs that same time.